### DISCUSSION TITLE
The renewal of the license on the site occurs with an error.

### DISCUSSION THREAD

**Discussion by denisgl7** [airsdk/Adobe-Runtime-Support#670] [General]
After entering the ID and filling out all the forms, registration ends at the third stage. I'm trying to renew my license through the website
![](https://i.ibb.co/Jd32v11/Screen-Recorder-Project12.gif)

**Comment by ajwfrost** [comment]
Hi @denisgl7  - can you please confirm what browser you're using, and I'll ask one of the team to investigate
thanks

**Comment by denisgl7** [comment]
Google chrome

пт, 29 янв. 2021 г. в 12:29, Andrew Frost <notifications@github.com>:

**Comment by denisgl7** [comment]
I tried Microsoft edge and Google chrome.I think it's not about the
browser.I was also a little late with the license renewal. I needed to
extend it until January 21. After that, I received a message about the
license extension for 10 days. Maybe the problem is in this period?


пт, 29 янв. 2021 г. в 13:56, Денис Гладкий <de.gladkiy@gmail.com>:

**Comment by ajwfrost** [comment]
Ah - yes there is some period where we allow a renewal, after that it needs a new license to be generated. I'll ask the folk to see if we can make a better error message! and also check whether this 10 day period could be extended...

thanks

**Comment by denisgl7** [comment]
Thanks!
But how can I renew my license right now? I have money, but the system does
not allow me to finish the purchase.

пт, 29 янв. 2021 г. в 15:28, Andrew Frost <notifications@github.com>:

**Reply by ajwfrost** [reply]
Currently you'd need to go through the process as if you were purchasing it for the first time..

**Comment by denisgl7** [comment]
Thank you for your quick answers! I'll do as you say

пт, 29 янв. 2021 г. в 15:39, Andrew Frost <notifications@github.com>:
